netqasm/sdk/builder.py

```diff
@@ -1269,6 +1299,83 @@ def undef_result_element(conn: BaseNetQASMConnection, _: RegFuture):
             loop_register=index_reg,
         )
 
+    def _build_cmds_epr_keep_corrections_single_pair(
+        self, bell_state: RegFuture, qubit_reg: operand.Register
+    ) -> None:
+        with bell_state.if_eq(1):  # Phi- -> apply Z-gate
```

Comment:
this 1, 2, 3 values, couldn't they be constants or an enum with a name?

Reply:
Using the `BellState` enum now

Reply:
You now use `BellState.PHI_MINUS.value` but `BellState.PHI_MINUS` is enough.

Reply:
`if_eq` actually requires an int (or more precisely, a `T_CValue`), so `value` is needed here